sgp4/model.py

```diff
@@ -89,6 +89,90 @@ def sgp4_array(self, jd, fr):
         r.shape = v.shape = len(jd), 3
         return e, r, v
 
+    def export_tle(self):
+        """Generate the TLE for the data in the `Satrec` object.
+
+        Forms the two string lines of the TLE and returns them as a tuple.
+
+         * ``line1``: line1 of the two-line-element set
+         * ``line2``: line2 of the two-line-element set
+        """
+
+        # Define constants
+        from math import pi
+        deg2rad = pi / 180.0  # 0.0174532925199433
+        xpdotp = 1440.0 / (2.0 * pi)  # 229.1831180523293
+
+        # --------------------- Start generating line 1 ---------------------
+
+        # Strings are immutable, so rather than replacing chars, we will build it
+        line1 = "1 "
+
+        # Pad the `satnum` entry with zeros
+        if len(str(self.satnum)) <= 5:
+            line1 += str(self.satnum).zfill(5)
```

Comment:
Building a longer and longer line tends to be inefficient. Instead, try creating a list `strings = []` and append to it all these little pieces of string, and then you can `''.join(strings)` at the end to all at once build a big string out of them.

Reply:
Thanks for the suggestion. I have now (locally) modified the string building methodology to this `append()` method. Will do the `str.format()` thing and recommit.